UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/Matrix.cs

```diff
@@ -539,6 +539,22 @@ public float GetRadiationLevel(Vector3Int localPosition)
 		return (0);
 	}
 
+	/// <summary>
+	/// Retrieves the local position from world position using UOP's OfficalPosition variable.
+	/// Avoid using this on every frame as it can generate a small amount of GC that can add up quickly.
+	/// </summary>
+	/// <param name="physics">the object's physics.</param>
+	/// <returns>The local position the object is on. Will return Vector3Int.ZERO if there's no Matrix assigned.</returns>
+	public static Vector3Int GetLocalPositionFromWorldPosition(UniversalObjectPhysics physics)
```

Comment:
There's a dictionary game object  to UniversalObjectPhysics, so you could make a version for this for-game object?

Reply:
wdym?

Reply:
as in it would be handy for game objects to, and it would avoid get component